tispark-overview.md

```diff
@@ -50,90 +65,106 @@ It is recommended to allocate 32G memory for Spark, and reserve at least 25% of
 
-```sh
-SPARK_EXECUTOR_CORES: 5
-SPARK_EXECUTOR_MEMORY: 10g
-SPARK_WORKER_CORES: 5
-SPARK_WORKER_MEMORY: 10g
-```
+To co-deploy TiKV and TiSpark, add TiSpark required resources to the TiKV reserved resources, and allocate 25% of the memory for the system.
 
-In the `spark-defaults.conf` file, add the following lines:
+### Deploy TiSpark with a standalone Spark cluster
 
-```sh
-spark.tispark.pd.addresses $your_pd_servers
-spark.sql.extensions org.apache.spark.sql.TiExtensions
-```
+For more information, see the [official configuration](https://spark.apache.org/docs/latest/spark-standalone.html) on the Spark website. 
 
-Add the following configuration in the `CDH` spark version:
+You are advised to use a standalone Spark cluster. To install Spark Standalone mode, you can simply place a compiled version of Spark on each node on the cluster. If you encounter any problem, see its [official website](https://spark.apache.org/docs/latest/spark-standalone.html). And you are welcome to [file an issue](https://github.com/pingcap/tispark/issues/new) on our GitHub.
 
-```
-spark.tispark.pd.addresses=$your_pd_servers
-spark.sql.extensions=org.apache.spark.sql.TiExtensions
-```
+#### Download and install
 
-`your_pd_servers` are comma-separated PD addresses, with each in the format of `$your_pd_address:$port`.
+You can download Spark 2.x from the [Apache Spark Archive](https://archive.apache.org/dist/spark/).
 
-For example, when you have multiple PD servers on `10.16.20.1,10.16.20.2,10.16.20.3` with the port 2379, put it as `10.16.20.1:2379,10.16.20.2:2379,10.16.20.3:2379`.
+For the Standalone mode without Hadoop support, use Spark **2.4.x** and any version of the pre-build binaries *with* Hadoop dependencies, for example `spark-2.4.8-bin-hadoop2.7.tgz`. If you need to use the Hadoop cluster, choose the corresponding Hadoop version. You can also choose to build from the [source code](https://spark.apache.org/docs/latest/building-spark.html) to match the previous version of the official Hadoop 2.x.
 
-### Configuration of hybrid deployment of TiKV and TiSpark
+Example:
 
-For the hybrid deployment of TiKV and TiSpark, add TiSpark required resources to the TiKV reserved resources, and allocate 25% of the memory for the system.
+```
+wget https://archive.apache.org/dist/spark/spark-2.4.8/spark-2.4.8-bin-hadoop2.7.tgz
+tar zxf spark-2.4.8-bin-hadoop2.7.tgz
+mv spark-2.4.8-bin-hadoop2.7 spark
+export SPARKPATH=~/spark # Also add this to your ~/.bashrc
+cd spark
+```
 
 ## Deploy the TiSpark cluster
 
-Download TiSpark's jar package [here](https://github.com/pingcap/tispark/releases). Download your desired version of jar package and copy the content to the appropriate folder.
+Download TiSpark's jar package [here](https://github.com/pingcap/tispark/releases) and place it in the `$SPARKPATH/jars` folder.
 
-### Deploy TiSpark on the existing Spark cluster
+```
+wget https://github.com/pingcap/tispark/releases/download/v2.4.1/tispark-assembly-2.4.1.jar
```

Comment:
For TiSpark-2.1.x, the jar name is quite different.
Shall we need to mention it?